toxcore/BUILD.bazel

```diff
@@ -1072,6 +1074,25 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "tox_types",
+    srcs = ["tox_types.c"],
+    hdrs = ["tox_types.h"],
+    visibility = ["//c-toxcore:__subpackages__"],
+)
+
+cc_library(
+    name = "tox_options",
+    srcs = ["tox_options.c"],
+    hdrs = ["tox_options.h"],
+    copts = ["-UTOX_HIDE_DEPRECATED"],
```

Comment:
is there a specific reason for undefining it?

Reply:
We need the declarations to check the definitions.